source/partials/schedule/_program.slim

```diff
@@ -0,0 +1,14 @@
+.Program
```

Comment:
isn't `program` a synonym of `schedule`?
I think using a different word that means exactly the same, just for referring to a different object only increases confusion